exporter/influxdbexporter/README.md

```diff
@@ -0,0 +1,141 @@
+# InfluxDB Exporter
+
+This exporter supports sending tracing, metrics, and logging data to [InfluxDB](https://www.influxdata.com/products/).
+
+Supported pipeline types: traces, metrics, logs
+
+## Configuration
+
+The following configuration options are supported:
+
+* `org` (Required): Org is the InfluxDB organization name of the destination bucket.
+* `bucket` (Required): Bucket is the InfluxDB bucket name that telemetry will be written to.
+* `protocol` (Optional): Protocol is the method used to deliver metrics to InfluxDB. Must be one of `line-protocol`, `grpc`. Defaults to `line-protocol`.
+* `line_protocol_server_url` (Required): LineProtocolServerUrl is the destination for Protocol `line-protocol`.
```

Comment:
Most exporters use `endpoint` setting as the destination URL. It would be useful to keep this uniform with the rest of exporters.